.github/workflows/clean.yml

```diff
@@ -0,0 +1,60 @@
+# This file was automatically generated by sbt-github-actions using the
+# githubWorkflowGenerate task. You should add and commit this file to
```

Comment:
Do we still need this file? If yes, we probably should remove this note as it is no longer auto-generated.

Reply:
i'd keep the file, as i think it's useful (if i understand correctly) and that header i kept as a way of giving credit to that plugin, but i can modify it so it's not misleading

Reply:
i added comment about the situation. in the future if we need to touch this file we can decide if we remove file, modify it by hand and update comment further, or regenerate it.